app/i18n/he/sub.php

```diff
@@ -127,12 +127,12 @@
 				'xpath' => 'XPath for:',	// TODO
 			),
 			'json_dotpath' => array(
```

Comment:
Should we rename the key `json_dotpath` too? (f.e. `json_dotnotation`?

Reply:
Yes, good idea

Reply:
Done.
But there are much more "dotpath" attributes/variables